internal/core/textlayout/sharedparley.rs

```diff
@@ -96,6 +96,9 @@ struct Brush {
     /// When set, this overrides the fill/stroke to use this color for just a fill, for selection.
     selection_fill_color: Option<crate::Color>,
     stroke: Option<TextStrokeStyle>,
+    underline_color: crate::Color,
+    strikethrough_color: crate::Color,
```

Comment:
In theory these could be part of the text decoration, but I think for now these colours should match the fill color of the text.